### PR TITLE
First Goodie Hack

### PR DESCRIPTION
**Pull request by gmahesh23**
**What does your Instant Answer do?**
It prints my Github Username and tells that I am awesome and completed the Goodie Hack.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It helps to find me among all other Github users.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://duck.co/duckduckhack/goodie_quickstart

**Why did you choose this data source?**
Because it is my first Goodie Hack.

**Are there any other alternative (better) data sources?**
No

**What are some example queries that trigger this Instant Answer?**
duckduckhack gmahesh23

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Github Users.

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot from 2015-07-06 19 03 21

**Checklist**

Please place an 'X' where appropriate.

- [] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack /design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

    - Linux (Ubuntu)
        - [x] Firefox

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/gmahesh23.pm [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package DDG::Goodie::IsAwesome::gmahesh23;
+# ABSTRACT: Write an abstract here
+# Start at https://duck.co/duckduckhack/goodie_overview if you are new
+# to instant answer development
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/gmahesh23.pm]: You can remove all comments except `# ABSTRACT:` which should read `gmahesh23's first Goodie`

File: lib/DDG/Goodie/IsAwesome/gmahesh23.pm [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+zci is_cached   => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "IsAwesome gmahesh23";
+description "Tell the world gmahesh23 is awesome ";
+primary_example_queries "first example query", "second example query";
+secondary_example_queries "optional -- demonstrate any additional triggers";
+
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/gmahesh23.pm";
+attribution github => ["GitHubAccount", "Friendly Name"],
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/gmahesh23.pm]: This can be `attribution github => ["https://github.com/gmahesh23", "gmahesh23"];`

**Comment by MrChrisW** [comment]
@gmahesh23 Thanks for submitting your first Goodie, I've added some feedback :+1:

**Comment by gmahesh23** [comment]
@MrChrisW  thanks for pointing out . I have made the changes . Do i need to send a new PR ?

File: lib/DDG/Goodie/IsAwesome/gmahesh23.pm [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+use DDG::Goodie;
+use strict;
+
+zci answer_type => "is_awesome_gmahesh23";
+zci is_cached   => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "IsAwesome gmahesh23";
+description "Tell the world gmahesh23 is awesome ";
+primary_example_queries "first example query", "second example query";
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/gmahesh23.pm]: mind removing these and the other comments? They're mostly for you to read and then remove ;)

**Comment by moollaza** [comment]
@gmahesh23 no need for a new PR. As you'll see, the new commit you pushed was automatically detected and added to this PR

**Comment by gmahesh23** [comment]
@moollaza I have removed the comments except the # Abstract .

**Comment by aleksandar-todorovic** [comment]
Looks good to me. Seems like this should be merged. Maybe the collaborators missed it?

**Comment by mintsoft** [comment]
@gmahesh23 @aleksandar-todorovic Apologies; this slipped through the cracks. We're actually not merging `isawesome` goodies in anymore as they've been phased out in favour of Cheatsheets which we think holds much more value.

If you're interested, they're really easy (https://duck.co/duckduckhack/goodie_cheat_sheets) and some suggestions for new sheets are here: https://duck.co/ideas

Again, apologies for the delay here.

**Comment by moollaza** [comment]
@gmahesh23 I just wanted to chime in here to apologize. During the course of your Pull Request, we made a decisions to stop accepting PR's that add IsAwesome Goodies to the repo, as they don't really add much value to DuckDuckGo for other users.

Seeing as we had asked you to make some update though, I don't mind letting this one in as an exception if you really want us to. Otherwise, as Mintsoft mentioned, we'd love for you to continue contributing by moving onto a cheat sheet which is considerably more valuable and can be used by everyone!

**Comment by gmahesh23** [comment]
It was a really long time since i did this ... but no problems. It would be
really good if this gets merged . Only if everything is proper and as per
your expectation . And I'm working on cheat sheets , I have already sent a
PR for that and will be discussing about that in that thread .

Thank you

**Comment by gmahesh23** [comment]
thank you .

**Pull request merged**